docs/reference/commandline/events.md

```diff
@@ -175,13 +175,17 @@ container container 588a23dac085 *AND* the event type is *start*
 
 The currently supported filters are:
 
+* config (`config=<name or id>`)
 * container (`container=<name or id>`)
 * daemon (`daemon=<name or id>`)
 * event (`event=<event action>`)
 * image (`image=<tag or id>`)
 * label (`label=<key>` or `label=<key>=<value>`)
 * network (`network=<name or id>`)
+* node (`node=<id>`)
 * plugin (`plugin=<name or id>`)
+* secret (`secret=<name or id>`)
+* service (`service=<name or id>`)
 * scope (`scope=<local or swarm>`)
```

Comment:
oh, I just notice, but I think this list is currently alphabetically ordered; so `secret` and `service` should go after `scope` 😅

Reply:
you are right. It is my mistake. updated.